app/views/teachers/otp/new.html.erb

```diff
@@ -7,9 +7,11 @@
   <%= f.hidden_field :uuid, value: @otp_form.uuid %>
   <%= f.hidden_field :email, value: @otp_form.email %>
 
-  <p class="govuk-body">We’ve emailed a confirmation code to <%= @otp_form.email %>.</p>
+  <p class="govuk-body">We’ve sent a login code to your email address <%= @otp_form.email %>.</p>
+  <p class="govuk-body">Use the code to sign into the Apply for QTS in England Service</p>
+  <p class="govuk-body">The code will expire after 15 minutes.</p>
 
-  <%= f.govuk_text_field :otp, size: "m", label: { text: "Enter your code", class: "govuk-label--s" }, class: "govuk-!-width-one-quarter" %>
+  <%= f.govuk_text_field :otp, size: "m", label: { text: "Enter code:", class: "govuk-label--s" }, class: "govuk-!-width-one-quarter" %>
```

Comment:
We don't use colons elsewhere in forms - is this something we definitely want?

Reply:
It is what is in the content changes, but i can double check with dom